Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php

```diff
@@ -68,9 +68,16 @@ public function __construct(
          * @return PropertyCollectionInterface Property values, indexed by their name
          */
         public readonly PropertyCollectionInterface $properties,
+        /**
+         * The node's name. The name is guaranteed to be for tethered nodes.
+         * For the regular classification, it can be set optionally.
+         */
         public readonly ?NodeName $nodeName,
         public readonly Timestamps $timestamps,
     ) {
+        if ($this->classification->isTethered() && $this->nodeName === null) {
+            throw new \InvalidArgumentException('The NodeName must be set if the Node is tethered.', 1695118377232);
```

Comment:
this ain't a unix timestamp 👁️ 
```suggestion
            throw new \InvalidArgumentException('The NodeName must be set if the Node is tethered.', 1695118377);
```

Reply:
lol then my tool is wrong - hold on 😱

Reply:
<img width="481" alt="image" src="https://github.com/neos/neos-development-collection/assets/85400359/5ff7687a-3b6c-45b2-b5f0-8fa4813c4c82">

hmmm this indeed adds three more digits 😭 

it has to be like this: https://www.in2code.de/en/recent/phpstorm-insert-unix-timestamp-in-php/
thank you for pointing this out 😅